keyboards/ymd96/keymaps/AnthonyWharton/keymap.c

```diff
@@ -22,7 +23,26 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define _DEFLT 0
 #define _RAISE 1
 
-#define KEYMAP KEYMAP_AnthonyWharton
+// Custom keymap for ISO Style Layout. Refer to the image in the README for
+// an image of the full layout
+#define KEYMAP( \
```

Comment:
I might have been wrong in telling you `config.h`, so that's my fault. It's not often when someone wants to add a layout/matrix to a keyboard. Sometimes they're fine, but other times the layouts are so esoteric that the user submitting them may be the only one who's going to use it.

Actually, I just now saw you had a readme file for your keymap. I usually ignore these because the user knows better than I do how their keymap works, and any issues will be apparent from the keymap itself. I see you've built your keyboard in a layout that matches [this image](https://i.imgur.com/SCzzBf2.png), but adapted for ISO layout. Your added matrix/layout should definitely go in `ymd96.h` exactly as it is, but defined as `LAYOUT_iso` rather than `KEYMAP`, and then the parts in your `keymap.c` that say `KEYMAP` (currently Lines 50 and 60) should both be changed to `LAYOUT_iso` as well.

Reply:
Brilliant ok, I'll get that done just now.